File: PHP/Barang.php
<?php
class Barang {
    private int $id = 0;
    private string $nama = "";
    private int $harga = 0;
    private int $stok = 0;
    private string $gambar = "";

    public function __construct(int $id = 0, string $nama = "", int $harga = 0, int $stok = 0, string $gambar = "") {
        $this->id = $id;
        $this->nama = $nama;
        $this->harga = $harga;
        $this->stok = $stok;
        $this->gambar = $gambar;
    }

    public function getId(): int { return $this->id; }
    public function setId(int $id) { $this->id = $id; }

    public function getNama(): string { return $this->nama; }
    public function setNama(string $nama) { $this->nama = $nama; }

    public function getHarga(): int { return $this->harga; }
    public function setHarga(int $harga) { $this->harga = $harga; }

    public function getStok(): int { return $this->stok; }
    public function setStok(int $stok) { $this->stok = $stok; }

    public function getGambar(): string { return $this->gambar; }
    public function setGambar(string $gambar) { $this->gambar = $gambar; }
}

File: Java/Barang.java
public class Barang {
    private int id;
    private String nama;
    private int harga;
    private int stok;

    public Barang() {
        this.id = 0;
        this.nama = "";
        this.harga = 0;
        this.stok = 0;
    }

    public Barang(int id, String nama, int harga, int stok) {
        this.id = id;
        this.nama = nama;
        this.harga = harga;
        this.stok = stok;
    }

    public int getId() { return id; }
    public void setId(int id) { this.id = id; }

    public String getNama() { return nama; }
    public void setNama(String nama) { this.nama = nama; }

    public int getHarga() { return harga; }
    public void setHarga(int harga) { this.harga = harga; }

    public int getStok() { return stok; }
    public void setStok(int stok) { this.stok = stok; }
}
